Tests/fixtures/podcasting.yml

```diff
@@ -9069,3 +9069,335 @@
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'YouTubeMusic/6.29.2 CFNetwork/1485 Darwin/23.1.0'
+  os:
+    name: iOS
+    version: "17.1"
+    platform: ""
+  client:
+    type: mobile app
+    name: Youtube Music
+    version: "6.29.2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo_Dot;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: smart speaker
+    brand: Amazon
+    model: 'Echo Dot'
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo_Dot_with_clock;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: smart speaker
+    brand: Amazon
+    model: 'Echo Dot'
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo_Show_5;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: tablet
+    brand: Amazon
+    model: 'Echo Show 5'
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'Amazon;Echo;27d4dfe427b34d57995b463e5d63198d;;tpapi;3.199.422'
+  os: [ ]
+  client:
+    type: mobile app
+    name: Alexa Media Player
+    version: ""
+  device:
+    type: smart speaker
+    brand: Amazon
+    model: Echo
+  os_family: Unknown
+  browser_family: Unknown
+
+-
+  user_agent: 'DS%20Podcast/2 CFNetwork/1410.0.3 Darwin/22.6.0'
+  os:
+    name: iOS
+    version: "16.6"
+    platform: ""
+  client:
+    type: mobile app
+    name: De Standaard
+    version: "2"
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'DS podcast/1.0.4 (be.standaard.audio; build:5; Android 11; Sdk:30; Manufacturer:samsung; Model: SM-A505FN) OkHttp/4.9.0'
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: mobile app
+    name: De Standaard
+    version: "1.0.4"
+  device:
+    type: smartphone
+    brand: 'Samsung'
+    model: 'Galaxy A50'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'SM-T970 (compatible; Tablet2.0) HandelsbladProduction, com.twipemobile.nrc 5.1.4 (511) / Android 33'
+  os:
+    name: Android
+    version: "33"
+    platform: ""
+  client:
+    type: mobile app
+    name: NRC
+    version: "5.1.4"
+  device:
+    type: "tablet"
+    brand: Samsung
+    model: 'Galaxy Tab S7+ 12.4" WiFi'
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Dalvik/2.1.0 (Linux; U; Android 13; SM-G991U Build/TP1A.220624.014) nprone_android/3.7.0/OSv:13'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: NPR
```

Comment:
Since other items were renamed to NPR One, is this one different somehow?

Reply:
The original 'NPR One' designation came from opawg2. The change to NPR also came from them. They reference this article, and indeed I cannot find any app any longer called NPR One on the android app store.

https://www.npr.org/about-npr/1218775096/npr-unifies-mobile-app-experience-for-listeners-sunsets-npr-one-app

Reply:
Ok, so old apps were NPR One, and new one is NPR, makes sense.